utils/k8s.go

```diff
@@ -66,3 +67,13 @@ func ReadK8sSecretAsString(namespace string, secretName string, secretKey string
 	}
 	return string(secretData), nil
 }
+
+// ReadCurrentK8sNamespace returns the value of the current namespace
+func ReadCurrentK8sNamespace() (string, error) {
+	contents, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		return "", fmt.Errorf("unable to read current namespace: %w", err)
+	}
+
+	return string(contents), nil
+}
```

Comment:
Would it be possible to just pass the current namespace via an enviornment variable instead?
```
            - name: K8S_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```